requirements.txt

```diff
@@ -1,3 +1,4 @@
 selenium == 3.0.0b2
```

Comment:
So FoxPuppet is a library, right? As what I have read in the past for those it's better to define dependencies in requirements.txt. Only end-user applications should define deps in setup.py and then most likely also in a strict way.